agoradatatools/etl/transform.py

```diff
@@ -50,12 +51,25 @@ def rename_columns(df: pd.core.frame.DataFrame, column_map: dict) -> pd.DataFram
 
     return df
 
+def nest_fields(df: pd.core.frame.DataFrame, grouping: str, new_column: str) -> pd.DataFrame:
```

Comment:
Whats the difference between `pd.core.frame.DataFrame`

Reply:
How silly.  They're the same.  It's really dependent on the way you import it.